src/net_processing.cpp

```diff
@@ -3890,6 +3891,12 @@ bool PeerLogicValidation::ProcessMessages(CNode* pfrom, std::atomic<bool>& inter
     }
     CNetMessage& msg(msgs.front());
 
+    TRACE4(PeerLogicValidation, ProcessMessages,
+           pfrom->addr.ToString().c_str(),
+           msg.m_command.c_str(),
+           MakeUCharSpan(msg.m_recv).data(),
```

Comment:
Why create a span just to take its data field?

Reply:
Think I copied this over from the peer logging PR and didn't simplify. Will fix.